src/node_modules.cc

```diff
@@ -344,6 +359,28 @@ void BindingData::GetNearestParentPackageJSONType(
   args.GetReturnValue().Set(Array::New(realm->isolate(), values, 3));
 }
 
+void BindingData::GetPackageJSONScripts(
+    const FunctionCallbackInfo<Value>& args) {
+  Realm* realm = Realm::GetCurrent(args);
+  std::string path = "package.json";
+
+  THROW_IF_INSUFFICIENT_PERMISSIONS(
+      realm->env(), permission::PermissionScope::kFileSystemRead, path);
+
+  auto package_json = GetPackageJSON(realm, path);
+  if (package_json == nullptr) {
+    printf("Can't read package.json\n");
```

Comment:
This seems to be used throughout... but I would say that doing printf that deep in the code is suboptimal. It would be better to return a string message to the user in case of errors... (This is not a request for change.)

Reply:
N